index.html

```diff
@@ -46,8 +47,175 @@ <h2 class="job-title">I'm a Software Developer</h2>
+                          has been the industry's standard</p>
+                         <ul class="cardb">
+                             <li class="bot button1">html</li>
+                             <li class="bot button2">bootstrap</li>
+                             <li class="bot button3">Ruby</li>
+                         </ul>
+                         <button class="link-button2" type="button">
+                             <a class="ltext1" href="#"> See Project </a>  
+                         </button> 
+                    </div>
+
+                     <div class="card card5">
+                        <title class="cardt" >Profesional Art Printing Data</title>
+                        <p class="cardp">A daily selection of privately personalized reads; no accounts or sign-ups required.
+                          has been the industry's standard</p>
+                         <ul class="cardb">
+                             <li class="bot button1">html</li>
+                             <li class="bot button2">bootstrap</li>
+                             <li class="bot button3">Ruby</li>
+                         </ul>
+                         <button class="link-button2" type="button">
+                             <a class="ltext1" href="#"> See Project </a>  
+                         </button> 
+                     </div>
+
+                     <div class="card card6">
+                        <title class="cardt" >Profesional Art Printing Data</title>
+                        <p class="cardp">A daily selection of privately personalized reads; no accounts or sign-ups required.
+                          has been the industry's standard</p>
+                         <ul class="cardb">
+                             <li class="bot button1">html</li>
+                             <li class="bot button2">bootstrap</li>
+                             <li class="bot button3">Ruby</li>
+                         </ul>
+                         <button class="link-button2" type="button">
+                             <a class="ltext1" href="#"> See Project </a>  
+                         </button> 
+                    </div>
+                </section>     
+        </section>
+
+        <!--About Me-->
+        <section class="About-myself">
+            <h1 class="about-text">About me</h1>
+            <p class="regular-text2">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+                 don’t hestiate to contact me.</p>
+             <button class="resume" type="button">Get My Resume</button>
+             <div class="line2"></div>
+             <div class="about-bar">
+                <nav class="bar">
+                    <div class="shape1"></div>
+                    <div class="titr">Language</div>
+                </nav>  
```

Comment:
- [ ] Great job using a `ul` and `li` tag to group the list of languages 👍. According to the project requirements pease use `ul` and `li` for categories in "About myself" section. `Languages`, `Frameworks`, and `Skills` 🙏

Reply:
@Ugboaja-Uchechi sorry I didn't understand your. should I put Language in UI tag and ruby and HTML and ... in li tag and to this for skills and the others, or i should put the languages and skills and framework in li tags?

Reply:
Hi @mohisa302, Your code should be something like a nested list. Example:

```
<ul>
  <li>
    Languages
    <ul>
      <li>
        Ruby
      </li>
    </ul>
  </li>
</ul>
```

I hope it's clearer for you.

Reply:
thanks so much

Reply:
done